internal/ingress/controller/controller.go

```diff
@@ -531,6 +531,11 @@ func (n *NGINXController) getBackendServers(ingresses []*ingress.Ingress) ([]*in
 				addLoc := true
 				for _, loc := range server.Locations {
 					if loc.Path == nginxPath {
+						// Same path but diffrent types are allowed
```

Comment:
I don't follow this comment. If the logic requires an explanation maybe move it into a helper function?